wdl/EvidenceQC.wdl

```diff
@@ -46,6 +46,8 @@ workflow EvidenceQC {
 
     Boolean run_ploidy = true
 
+    Float? melt_insert_size
```

Comment:
This would be an Array[Float?]? because it would be one insert size value for each sample